po/fr.po

```diff
@@ -0,0 +1,244 @@
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#: src/UI/MainWindow.vala:43
+msgid "Not saved yet"
+msgstr "Pas encore enregistré"
+
+#: src/UI/MainWindow.vala:138 src/UI/MainWindow.vala:401
+msgid "Spreadsheet"
+msgstr "Spreadsheet"
+
+#: src/UI/MainWindow.vala:138
+msgid "Start something new, or continue what you have been working on."
+msgstr "Commencez quelque chose de nouveau, ou continuez votre travail."
+
+#: src/UI/MainWindow.vala:139
+msgid "New Sheet"
+msgstr "Nouvelle feuille"
+
+#: src/UI/MainWindow.vala:139
+msgid "Create an empty sheet"
+msgstr "Créer une feuille vierge"
+
+#: src/UI/MainWindow.vala:140
+msgid "Open File"
+msgstr "Ouvrir un fichier"
+
+#: src/UI/MainWindow.vala:140
+msgid "Choose a saved file"
+msgstr "Sélectionner un fichier enregistré"
+
+#: src/UI/MainWindow.vala:141
+msgid "Open Last File"
+msgstr "Ouvrir le dernier fichier"
+
+#: src/UI/MainWindow.vala:141
+msgid "Continue working on foo.xlsx"
+msgstr "Continuer à travailler sur foo.xlsx"
+
+#: src/UI/MainWindow.vala:147 src/UI/MainWindow.vala:323
+#: src/UI/MainWindow.vala:461
+msgid "Open a file"
+msgstr "Ouvrir un fichier"
+
+#: src/UI/MainWindow.vala:148 src/UI/MainWindow.vala:324
+#: src/UI/MainWindow.vala:357
+msgid "_Cancel"
+msgstr "_Annuler"
+
+#: src/UI/MainWindow.vala:150 src/UI/MainWindow.vala:326
+msgid "_Open"
+msgstr "_Ouvrir"
+
+#: src/UI/MainWindow.vala:155 src/UI/MainWindow.vala:331
+#: src/UI/MainWindow.vala:364
+msgid "CSV files"
+msgstr "Fichiers CSV"
+
+#: src/UI/MainWindow.vala:184
+msgid "Insert functions to a selected cell"
+msgstr "Insérer des fonctions dans la cellule sélectionnée"
+
+#: src/UI/MainWindow.vala:186
+msgid "Click to insert numbers or functions to a selected cell"
+msgstr ""
+"Cliquez pour insérer des nombres ou des fonctions dans la cellule "
+"sélectionnée"
+
+#: src/UI/MainWindow.vala:217
+msgid "Search functions"
+msgstr "Rechercher des fonctions"
+
+#: src/UI/MainWindow.vala:244
+msgid "Set colors to letters in a selected cell"
+msgstr "Définir la couleur de police de la cellule sélectionnée"
+
+#: src/UI/MainWindow.vala:301
+#, c-format
+msgid "Untitled Spreadsheet %i"
+msgstr "Classeur sans titre %i"
+
+#: src/UI/MainWindow.vala:312
+msgid "Page 1"
+msgstr "Feuille 1"
+
+#: src/UI/MainWindow.vala:356
+msgid "Save your work"
+msgstr "Enregistrez votre travail"
+
+#: src/UI/MainWindow.vala:359
+msgid "_Save"
+msgstr "_Enregistrer"
+
+#: src/UI/MainWindow.vala:453
+msgid "Create a new empty file"
+msgstr "Créer un nouveau fichier vierge"
+
+#: src/UI/MainWindow.vala:469
+msgid "Save this file with a different name"
+msgstr "Enregistrer ce fichier avec un nom différent"
+
+#: src/UI/MainWindow.vala:477
+msgid "Redo"
+msgstr "Rétablir"
+
+#: src/UI/MainWindow.vala:485
+msgid "Undo"
+msgstr "Annuler"
+
+#: src/Widgets/StyleModal.vala:12
+msgid "Fonts"
+msgstr "Police"
+
+#: src/Widgets/StyleModal.vala:13
+msgid "Cells"
+msgstr "Cellule"
```

Comment:
Probably missing a "s".

Reply:
I don't think so because here the plural is used to generalize something. In French it should be singular.

In the context of the app the "Fonts" and the "Cells" attribute are applied on one specific cell si there is no reason to use plural form.

Plus "Police" which is the translation of "Font" is never plural on French.

Reply:
> In the context of the app the "Fonts" and the "Cells" attribute are applied on one specific cell si there is no reason to use plural form.

Should we update the original strings to singular ones then?

Reply:
In my opinion this looks better but I'm not sure

Reply:
Created a PR: #67